app/login/page.tsx

```diff
@@ -89,7 +117,9 @@ const Login = () => {
                   <FormControl>
                     <Input
                       placeholder="Enter your email address"
-                      className="h-11 rounded border-none bg-black-800 paragraph-3-medium"
+                      className={`h-11 rounded placeholder:font-normal border-[1px] dark:border-none border-gray-300/40 ${
+                        mode === "dark" ? "bg-black-800" : "bg-white-100"
```

Comment:
The entire point of having a dark mode in tailwind is that the CSS variables are changed. This means that you should never have to do this, instead in the global.css file you can make `black-800` actually be white. This is why it's not ideal to call the variables black, because on light mode, it does not make sense :)